reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,68 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
+Therefore, PowerShell Core is not included in the Windows 10 or Windows Server licensing agreements.
+
+However, PowerShell Core is supported under traditional Microsoft support agreements, including [Premier][] and [Microsoft Enterprise Agreements][enterprise-agreement].
+You can also pay for [assisted support][] for PowerShell Core by filing a support request for your problem.
+
+We also offer [community support][] on GitHub where you can file an issue, bug, or feature request.
+We offer no guarantee there that your issue will be addressed or resolved in a timely manner.
+If you have a problem that requires immediate attention,
+you should use the traditional, paid support options.
+
+## Lifecycle of PowerShell Core
+
+PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern].
+This support lifecycle is intended to keep customers up-to-date with the latest versions.
+
+The version 6.x branch of PowerShell Core will be updated approximately once every six months (e.g. 6.0, 6.1, 6.2, etc.)
+
+> [!IMPORTANT]
+> You must update within six months after each new minor version release to continue receiving support.
+
+For example, if PowerShell Core 6.1 is released on July 1st, 2018,
+you would be expected to update to PowerShell Core 6.1 by January 1st, 2019 to maintain support.
+
+![PowerShell Core branch lifecycle][lifecycle-chart]
+
+Eventually, we expect PowerShell Core will adopt the "long-term servicing" approach where we would require only servicing and security updates to stay in support on a specific branch/version of 6.x.
+
+## Notes on licensing
+
+PowerShell Core is released under the [MIT license][].
+Under this license, and in the absence of a paid support agreement,
+users are limited to [community support][].
+With community support, Microsoft makes no guarantees of responsiveness or fixes.
+
+## Windows PowerShell Module
+
+Support for PowerShell Core does not extend to other product modules unless those modules explicitly support PowerShell Core.
+For example, using the `ActiveDirectory` module that ships as part of Windows Server is an unsupported scenario.
+
+However, modules that do not explicitly support PowerShell Core may be compatible in some cases.
```

Comment:
I'd remove the technical details below.

Reply:
In order to drive people to push on compatibility problems, I've been asked to make that sort of info as highly visible as possible. It probably should live in another document though, you're right. I'm going to leave it for now, and remove it once I have something in official docs to link to. 

Reasonable?

Reply:
Yes.

Reply:
We should probably have an entire doc on Windows PowerShell compatibility and just reference that doc from this one.